test/test_distributions.py

```diff
@@ -242,6 +242,14 @@ def test_log_prob(jax_dist, sp_dist, params, prepend_shape, jit):
         expected = sp_dist.logpdf(samples)
     except AttributeError:
         expected = sp_dist.logpmf(samples)
+    except ValueError as e:
+        # precision issue: np.sum(x / np.sum(x)) = 0.99999994 != 1
```

Comment:
Maybe we should lower this precision value in the transform or have it be dependent on the data, since I think scipy would do the normalization behind the scenes anyways.

Reply:
This is indeed scipy problem. Scipy works in float64 so their check is `< 1e-10` would be fine for the output of `np.sum(x / np.sum(x))`. Our check `1e-6` in float32 should be fine but our samples would not be valid samples for scipy.